gravitee-apim-console-webui/src/management/api/documentation-v4/api-documentation-v4.module.ts

```diff
@@ -121,6 +123,8 @@ import { GioApiMetadataListModule } from '../component/gio-api-metadata-list/gio
     ApiDocumentationV4BreadcrumbComponent,
     ApiDocumentationV4FileUploadComponent,
     ApiDocumentationV4PageTitleComponent,
+    ApiDocumentationChoosePageListComponent,
+    ApiDocumentationChooseExistingPageComponent,
```

Comment:
Why do you need to declare them in the module? If they are standalone, I believe that they don't need to be included in any modules.